recipes/nvtx/all/conanfile.py

```diff
@@ -0,0 +1,35 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.35.0"
+
+class NVTXConan(ConanFile):
+    name = "nvtx"
+    description = "The NVIDIA Tools Extension SDK (NVTX) is a C-based API for annotating events, code ranges, and resources in your applications."
+    homepage = "https://github.com/NVIDIA/NVTX"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    topics = ("profiler", "nvidia", "nsight")
+    settings = "os"
```

Comment:
```suggestion
    settings = "os", "arch", "compiler", "build_type"
```

Not sure, butI think it will be required for conan v2